tsconfig.json

```diff
@@ -6,6 +6,7 @@
         "rootDir": "src",
         "outDir": "build",
         "allowJs": true,
+        "pretty": true,
```

Comment:
oh yeah, this just provides some extra colours when you see typescript errors in the console.